src/solution.jl

```diff
@@ -1,13 +1,10 @@
 function add_variables!(model, var::AbstractVariable)
-    var.id_hash == objectid(:constant) &&
-        error("Internal error: constant used as variable")
```

Comment:
No idea why this is here. Safe to remove? The comparison seems arbitrary and unrelated to the actual `id_hash` of a constant.